pkg/rpcclient/rpc_test.go

```diff
@@ -1986,7 +1990,7 @@ func wrapInitResponse(r *params.In, resp string) string {
 	var response string
 	switch r.Method {
 	case "getversion":
-		response = `{"id":1,"jsonrpc":"2.0","result":{"protocol":{"network":42},"tcpport":20332,"wsport":20342,"nonce":2153672787,"useragent":"/NEO-GO:0.73.1-pre-273-ge381358/"}}`
+		response = `{"id":1,"jsonrpc":"2.0","result":{"protocol":{"network":42},"tcpport":20332,"wsport":20342,"nonce":2153672787,"useragent":"/NEO-GO:0.73.1-pre-273-ge381358/", "rpc": {"maxiteratorresultitems": 100, "sessionenabled": false}}}`
```

Comment:
What I meant is that we should adjust the following testcase: https://github.com/nspcc-dev/neo-go/blob/be1b97d04e416d110de63b8f4ddd817dd8363c9d/pkg/rpcclient/rpc_test.go#L1021

But this adjustment is also valid, let's keep it.

Reply:
they seem to be connected